python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/search.py

```diff
@@ -223,19 +269,19 @@ def _parse_dataset(self, entity: dict[str, Any], matches) -> SearchResult:
         tags = parse_tags(entity)
         last_updated = parse_last_updated(entity)
         name = entity["name"]
-        relationships = entity.get("relationships", {})
-        total_data_products = relationships.get("total", 0)
-        data_products = relationships.get("relationships", [])
-        data_products = [
-            {"id": i["entity"]["urn"], "name": i["entity"]["properties"]["name"]}
-            for i in data_products
-        ]
+
+        relations_dict = parse_relations(entity.get("relationships", {}))
 
         metadata = {
             "owner": owner_name,
             "owner_email": owner_email,
-            "total_data_products": total_data_products,
-            "data_products": data_products,
+            "total_parents": relations_dict["total"],
+            "parents": relations_dict["entities"],
```

Comment:
I think you are right we might have to change this if we end up using data products and containers together, but we might not need the distinction so keeping it simple makes sense for now